packages/expo-development-client/ios/EXDevelopmentClientController.m

```diff
@@ -102,6 +103,71 @@ - (void)navigateToLauncher {
   [_window makeKeyAndVisible];
 }
 
-@end
+- (BOOL)onDeepLink:(NSURL *)url options:(NSDictionary *)options {
+  if (![url.host isEqual:@"open-app"]) {
```

Comment:
can we change this to "expo-development-client"

Reply:
yes, we can :D